lib/manager/pip_requirements/extract.js

```diff
@@ -11,8 +11,23 @@ module.exports = {
   dependencyPattern,
   packagePattern,
   extractPackageFile,
+  isSkipComment,
```

Comment:
I'm wondering if this logic should logically exist independent of pip_requirements and instead be in some (new) python helper package

Reply:
Could we include this in all package managers?

Reply:
Yes. This syntax or something similar could be supported by additional package managers, including non-python